components/Grid/Grid.vue

```diff
@@ -181,10 +184,26 @@ export default {
             'setDraggedElement',
             'setDraggedElIndex',
             'setGhostElXTranslation',
-            'setBounds',
-            'setGhostElTransform',
             'setGhostIndex',
+            'setGhostFilterIndex',
         ]),
+        onMouseOverGrid(isOver) {
+            this.isMouseOverGrid = isOver;
+        },
+        onDragLeave({ pageX, pageY }) {
```

Comment:
it looks very similar to components/Grid/AdvancedFilters/GridAdvancedFiltersContainer.vue onDragLeave method